app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,481 @@ def navigation
+              items: [
                 {
-                  :text   => "Scala museum, Milan",
-                  :url    => "#",
-                  :number => '1234' + ' ' + t('site.channels.data-types.count'),
-                  :date   => "July 2015"
+                  text: "Scala museum, Milan",
+                  url: "#",
+                  number: '1234' + ' ' + t('site.channels.data-types.count'),
+                  date: "July 2015"
                 },
                 {
-                  :text   => "Dresden Library",
-                  :url    => "#",
-                  :number => '4377' + ' ' + t('site.channels.data-types.count'),
-                  :date   => "November 2014"
+                  text: "Dresden Library",
+                  url: "#",
+                  number: '4377' + ' ' + t('site.channels.data-types.count'),
+                  date: "November 2014"
                 },
                 {
-                  :text   => "Rijksmuseum, Amsterdam",
-                  :url    => "#",
-                  :number => '2169' + ' ' + t('site.channels.data-types.count'),
-                  :date   => "September 2014"
+                  text: "Rijksmuseum, Amsterdam",
+                  url: "#",
+                  number: '2169' + ' ' + t('site.channels.data-types.count'),
+                  date: "September 2014"
                 }
               ],
-              :more_link  => {
-                :text  => "Load more",
-                :url   => "#777"
+              more_link: {
+                text: "Load more",
+                url: "#777"
               }
             },
-            :credits => {
-              :title => t('site.channels.labels.credits'),
-              :items => [
+            credits: {
+              title: t('site.channels.labels.credits'),
+              items: [
                 {
-                  :text  => "Deutsche National Bibliothek",
-                  :url   => "#777"
+                  text: "Deutsche National Bibliothek",
+                  url: "#777"
                 },
                 {
-                  :text  => "Netherlands institute for sound and vision",
-                  :url   => "#777"
+                  text: "Netherlands institute for sound and vision",
+                  url: "#777"
                 },
                 {
-                  :text => "Cité de la Musique",
-                  :url => "#777"
+                  text: "Cité de la Musique",
+                  url: "#777"
                 }
               ]
             }
           },
-          :hero_config =>{
-            :hero_image => "sample/channel_hero_music.jpg",
-            :attribution_text => "This is a placeholder image found somewhere",
-            :attribution_url => "http://europeana.eu",
-           # :license_CC-ND-NC-SA => true
-            :license_CC_ND_NC_SA => true
+          hero_config: {
+            hero_image: "sample/channel_hero_music.jpg",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.